### ISSUE TITLE
My Stuff

### ISSUE THREAD

**Issue by BobRoman** [pixaura/cliqrex-main#176]
### Avoid Duplicates

- [X] I have searched the [issue tracker](https://github.com/pixaura/cliqrex-main/issues), without success.

### Description

When a user clicks on the "My Stuff" in the lower navigation, this is the page that appears. The user can share their profile, edit their profile, and manage their lists.

### Acceptance Criteria

- My Profile Card on the top of the My Stuff Page contains, View/Change Image, View Name, and User Name, Gear Icon (the ability to edit user settings), Share Profile Icon (the ability to share the user's profile)
- Lists to include are: In Progress, Up Next (the pre-built list), Custom Lists, Followed Lists, and Top Rated (TV Shows, Movies, and Podcasts)

**Custom Lists**
- Custom List Card includes: Title, Gear Icon (Settings), Share Icon, Locked/Unlocked (for private or public), and More
- "More" link goes to the page view of all the media within the list.
- In Custom Lists, there's a button to add "+ New List" This will open up the Edit List Modal Dialogue

**Followed Lists**
- Followed List Card Includes: Title, Starred icon, Share icon, Media Items, More Link
- Can unfollow a list by clicking the yellow star
- "More" link goes to the page view of all the media within the list.

**Top Rated Lists**
- Top Rated Lists include: TV Shows, Movies, and Podcasts
- Top Rated Card Includes: Title, Share icon, Media Items, More Link
- "More" link goes to the page view of all the media within the list.

**Dropped and Finished Buttons**
- Clicking Dropped will take you to a list displaying media items with the status "Dropped"
- Clicking Finished will take you to a list displaying media items with the status "Finished"

### Exceptions

none

### Screenshots

![My_Stuff](https://user-images.githubusercontent.com/15748883/179512085-b1883d19-1668-4799-9242-6abc95360ae3.png)

**Comment by DaraOloughlin** [comment]
Unable to test 176 as My Stuff screen is all grey. Bug report created in Trello.

**Comment by DaraOloughlin** [comment]
@BobRoman did you submit to @palepinkdot the bug report I put in Trello about my "My Stuff" being all grey? It's been all grey and nonfunctional for nine days, yet I see this issue is closed.

**Comment by palepinkdot** [comment]
> @BobRoman did you submit to @palepinkdot the bug report I put in Trello about my "My Stuff" being all grey? It's been all grey and nonfunctional for nine days, yet I see this issue is closed.

Since its not a separate issue I only did preliminary testing on it. Was not able to reproduce on several accounts nor was I able to make it happen outside of  a single time, which was fixed by updating the Google OA URL because it changes sporadically. After poking around I cannot determine what caused the grey screen outside of a mishandled user edge case.
